.github/workflows/release-java-provider.yml

```diff
@@ -39,4 +50,4 @@ jobs:
         uses: goreleaser/goreleaser-action@v6
```

Comment:
Shouldnt this copy the check-release step from https://github.com/pulumi/pulumi-dotnet/blob/main/.github/workflows/release-dotnet-provider.yml as well?